app/src/main/java/com/simplecity/amp_library/ui/views/multisheet/CustomMultiSheetView.java

```diff
@@ -22,8 +17,8 @@
 
     private static final String TAG = "CustomMultiSheetView";
 
-    @Inject MultiSheetEventRelay multiSheetEventRelay;
-    @Inject MultiSheetSlideEventRelay multiSheetSlideEventRelay;
+    MultiSheetEventRelay multiSheetEventRelay = new MultiSheetEventRelay();
+    MultiSheetSlideEventRelay multiSheetSlideEventRelay = new MultiSheetSlideEventRelay();
```

Comment:
We should not be instantiating the `MultiSheetSlideEventRelay()` here. Since it's an event really, there should only be one instance across the app. This is instantiated in the `MultiSheetModule` in order to inject it elsewhere.